src/org/openstreetmap/josm/plugins/pt_assistant/validation/PTAssistantValidatorTest.java

```diff
@@ -228,18 +226,11 @@ public void run() {
      *             if interrupted
      */
     private int showIncompleteMembersDownloadDialog() throws InterruptedException {
+        return PTProperties.DOWNLOAD_INCOMPLETE.get() ? JOptionPane.YES_OPTION : JOptionPane.NO_OPTION;
 
-        if (Main.pref.getBoolean("pt_assistant.download-incomplete", false) == true) {
-            return JOptionPane.YES_OPTION;
-        }
-
-        if (Main.pref.getBoolean("pt_assistant.download-incomplete", false) == false) {
-            return JOptionPane.NO_OPTION;
-        }
-
-        IncompleteMembersDownloadDialog incompleteMembersDownloadDialog = new IncompleteMembersDownloadDialog();
-        return incompleteMembersDownloadDialog.getUserSelection();
-
+        // TODO: The following is dead code! Either revive it or throw it away.
+        // IncompleteMembersDownloadDialog incompleteMembersDownloadDialog = new IncompleteMembersDownloadDialog();
+        // return incompleteMembersDownloadDialog.getUserSelection();
```

Comment:
This addresses the problem noted in https://github.com/JOSM/pt_assistant/commit/e7fe541d2ea286b8cdab9b5b6f7ca0e7fca1b621#r30402943

The code that is commented out and the classes `IncompleteMembersDownloadDialog` and `ProceedDialog` should be revisited by someone more familiar with the plugin, if they are still needed.